apps/web/components/eventtype/InstantEventController.tsx

```diff
@@ -97,3 +99,193 @@ export default function InstantEventController({
     </LicenseRequired>
   );
 }
+
+const InstantMeetingWebhooks = ({ eventType }: { eventType: EventTypeSetup }) => {
+  const { t } = useLocale();
+  const utils = trpc.useContext();
+  const { data: webhooks } = trpc.viewer.webhook.list.useQuery({ eventTypeId: eventType.id });
```

Comment:
Why do we want to list all webhooks in the Instant Booking tab?

Reply:
Oh right. forgot to update this